src/flb_utils.c

```diff
@@ -99,11 +105,23 @@ void flb_utils_error(int err)
         fprintf(stderr,
                 "%sError%s: undefined. Aborting",
                 ANSI_BOLD ANSI_RED, ANSI_RESET);
+        #ifdef FLB_HAVE_AWS_ERROR_REPORTER
```

Comment:
please align ALL the pre-processor conditions to the left and we should be ready to go.

Reply:
ACK, will send out a new PR to fix this.